xhyve/xhyve.go

```diff
@@ -414,7 +414,7 @@ func (d *Driver) PreCreateCheck() error {
 	if err != nil {
 		return fmt.Errorf("Error detecting VBox version: %s", err)
 	}
-	if !strings.HasPrefix(ver, "5") {
+	if strings.HasPrefix(ver, "4") {
 		return fmt.Errorf("Virtual Box version 4 or lower will cause a kernel panic" +
 			"if xhyve tries to run. You are running version: " +
```

Comment:
Gah, shoulda fixed the spacing typo between "panic" and "if" while you were in there!

Reply:
FYI : Doesn't work for version 6.0.4.

Error with pre-create check: "Virtual Box version 4 or lower will cause a kernel panicif xhyve tries to run. You are running version: 6.0.4r128413\n\n\t Please upgrade to version 5 at https://www.virtualbox.org/wiki/Downloads"